lib/statsd.rb

```diff
@@ -165,28 +220,28 @@ def select_shard(stat)
     end
   end
 
-  def signed_payload(key, message)
-    sha256 = Statsd.setup_openssl
-    payload = timestamp + nonce + message
-    signature = OpenSSL::HMAC.digest(sha256, key, payload)
-    signature + payload
-  end
+  class Buffer
+    DEFAULT_BUFFER_CAP = 512
```

Comment:
...However, if you think it'd be easier for other people in the future, I can copy paste the commit message as a comment.